coalib/collecting/Collectors.py

```diff
@@ -154,7 +154,7 @@ def collect_bears(bear_dirs, bear_globs, kinds, log_printer,
                                 bears.
     :param bear_globs:          Globs of bears to collect.
     :param kinds:               List of bear kinds to be collected.
-    :param log_printer:         LogPrinter to handle logging.
+    :param log_printer:         log_printer to handle logging.
```

Comment:
there are other functions in the same file having  log_printer as parameter;  
they all have the same param docs for log_printer that is why i modified LogPrinter to have something uniform